.azure-pipelines/all.yml

```diff
@@ -15,9 +15,6 @@ jobs:
     vmImage: "Ubuntu-16.04"
   strategy:
     matrix:
-      Rb23:
```

Comment:
Why are we removing this one? Is it redundant?

Reply:
Not redundant but AZP removed this version of Ruby from their agents. Looks like there has been a `Note` on their release pages for a bit but the latest ubuntu-1604 image (released earlier this month) doesn't include Ruby 2.3.x https://github.com/microsoft/azure-pipelines-image-generation/releases/tag/ubuntu16%2F20191202.1

Reply:
Due to ^ the latest build on master failed actually - https://dev.azure.com/datadoghq/dogapi-rb/_build/results?buildId=4590&view=logs&j=a352be1d-4b0d-5db4-a38c-6829b6c2207b&t=9f773940-193d-5f4b-2747-05c376291b32 (That log confirms these are the only three Ruby versions available on the image)